lib/area.js

```diff
@@ -13,8 +13,14 @@ const basicConfig = {
       'blockquote',
       'alignLeft',
       'alignCenter',
-      'alignRight'
+      'alignRight',
+      'codeBlock',
+      'table',
+      'image',
+      'undo',
+      'redo'
```

Comment:
Brought in some other stuff that was in the assembly demo. If it's too much clutter let me know and I'll remove it.